src/pages/user/libraries/[[...id]].tsx

```diff
@@ -32,12 +40,20 @@ const LibrariesHome: NextPage<ILibrariesHomeProps> = ({ id, subpage, from }) =>
     { enabled: !!id, staleTime: 0 },
   );
 
+  useEffect(() => {
+    if (library) {
+      setLibrary(library);
```

Comment:
I think actually we shouldn't do this.  Instead of using a store or prop drilling just refetch it in the child components.  Anything that we can get from the server, we should use react-query. 

Could definitely use this store though to manage the state for `onNoteUpdated` or `selectedBibcodes`, for example, which are currently getting passed down through.